File: testprojects/master_python_mainone/pipelines/PYTHON_DEP_MANAGEMENT_ALL/code/job/graph/SubGraph_7/Reformat_8.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from job.udfs.UDFs import *

def Reformat_8(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("c_int_new"))

File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/Reformat_8.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_8 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(
      col("customer_id"),
      col("first_name"),
      col("last_name"),
      col("phone"),
      col("email"),
      col("country_code"),
      col("account_open_date"),
      col("account_flags"),
      lookup("Lookup_1", col("customer_id"), col("first_name"))
        .getField("email")
        .as("lookup1")
    )

}
